src/components/Property/RaiseRepair/RaiseRepairForm.js

```diff
@@ -25,11 +26,15 @@ const RaiseRepairForm = ({
   trades,
   contacts,
   onFormSubmit,
+  currentUserRaiseLimit,
```

Comment:
Wonder if we just call this `raiseLimit` as this component perhaps doesn't need to know where it's from...